docker-compose.yml

```diff
@@ -1,39 +1,65 @@
-version: "3"
+version: "2"
```

Comment:
Why are we downgrading?

Reply:
Balena is work with this version.
`docker-compose.dev.yml` uses `version: 3`

Reply:
Got it. Fair.